games-rpg/baldurs-gate-ii-ee/baldurs-gate-ii-ee-2.6.6.0.47292.ebuild

```diff
@@ -0,0 +1,52 @@
+# Copyright 1999-2023 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+inherit desktop unpacker wrapper
+
+DESCRIPTION="Baldur's Gate II: Enhanced Edition"
+HOMEPAGE="https://baldursgate.beamdog.com/"
+SRC_URI="baldur_s_gate_ii_enhanced_edition_${PV//./_}.sh"
+
+LICENSE="GOG-EULA"
+SLOT="0"
+KEYWORDS="-* ~amd64"
+RESTRICT="bindist fetch"
```

Comment:
You often add `splitdebug` here for proprietary games since they generally don't include debug symbols.

Reply:
Done, added `xdg` too, thanks. If I update the `baldurs-gate-ee` ebuild in the future I'll match it to this too.